user/articles/workflow/03-workflow-metrics-reports.markdown

```diff
@@ -61,7 +61,7 @@ report and you'll see a more detailed table including the following columns:
 **ID**
 : This is the workflow item's numeric identifier to the system. Importantly, you
 can click this to enter the Item Detail view. If you're doing something
-programmatically, this represents the
+programmatically, this represents the instance.
```

Comment:
@inacionery I think we should just delete this last sentence.

> If you're doing something programmatically, this represents the instance.

I don't think it's helpful to say this here in the User docs. Let me know if you disagree.

Reply:
ok, we could remove it.